openlineplanner/src/components/MapStationPopup.vue

```diff
@@ -189,7 +189,7 @@ export default {
   // align-items: center;
   justify-content: center;
   flex-direction: column;
-  z-index: 4;
+  z-index: 10;
```

Comment:
Did it fix any issue? I know there are still some layover bugs occurring.. lets fix them soon

Reply:
If thats extra work.. no need to. It will be changed anyway I guess..